package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "prepublish": "npm ls",
     "install": "node-pre-gyp install --fallback-to-build",
-   "test": "node-pre-gyp install --fallback-to-build && standard && tap test.js"
+    "test": "node-pre-gyp install --build-from-source && standard && tap test.js"
```

Comment:
Changed this command to ensure when running tests it always rebuilds. Previously it used any existing build, which wasn't optimal in dev.